tests/framework/utils/k8s_helper.go

```diff
@@ -1422,42 +1454,77 @@ func (k8sh *K8sHelper) getContainerLogs(podAppName, hostType, namespace, testNam
 	}
 
 	for _, pod := range podList.Items {
-		podName := pod.Name
-		logger.Infof("getting logs for pod : %v", podName)
-		res := k8sh.Clientset.CoreV1().Pods(namespace).GetLogs(podName, logOpts).Do()
-		rawData, err := res.Raw()
-		if err != nil {
-			logger.Errorf("Cannot get logs for app : %v in namespace %v, err: %v", podName, namespace, err)
-			continue
-		}
-		dir, _ := os.Getwd()
-		fpath := path.Join(dir, "_output/tests/")
-		if _, err := os.Stat(fpath); os.IsNotExist(err) {
-			err := os.MkdirAll(fpath, 0777)
-			if err != nil {
-				logger.Errorf("Cannot get logs files dir for app : %v in namespace %v, err: %v", podName, namespace, err)
-				continue
-			}
-		}
-		logSuffix := ""
-		if containerName != "" {
-			logSuffix = "_" + containerName
-		}
-		fileName := fmt.Sprintf("%s_%s_%s_%s%s_%d.log", testName, hostType, podName, namespace, logSuffix, time.Now().Unix())
-		fpath = path.Join(fpath, fileName)
-		file, err := os.Create(fpath)
-		if err != nil {
-			logger.Errorf("Cannot get logs files for app : %v in namespace %v, err: %v", podName, namespace, err)
-			continue
-		}
+		k8sh.getPodLogs(pod, platformName, namespace, testName, previousLog)
```

Comment:
I'm a bit confused here. It seems backwards that `getContainerLogs` calls `getPodLogs` when pods have containers. Would it be cleaner just to have a `getPodLogs` method and leave it at that?

Reply:
Good catch, I need to finish cleaning this up. There is only one caller for this method now so I can just merge them and remove the confusion.